Contributing.md

```diff
@@ -82,4 +82,23 @@ to the TC and the TC uses its standard consensus seeking process to evaluate whe
 not to add this new member. Members who do not participate consistently at the level of
 a majority of the other members are expected to resign.
 
-
+# Project Captains
+
+The Express TC can designate captains for individual projects/repos in the
+three organizations. These captains are responsible for being the primary
+day-to-day maintainers of the repo on a technical and community front.
+Repo captains are empowered with repo ownership and module publication rights.
+When there are conflicts, especially on topics that effect the Express project
+at large, captains are responsible to bring it up with the TC and drive
+those conflicts to resolution. Captains are also responsible in making sure
+community members follow the community guidelines, maintaining the repo
+and the registry, as well as in providing user support.
+
+## Current Project Captains
```

Comment:
Do we just want to add the volunteers here now? And then have the final ratification be approval of this PR?

Reply:
added the volunteers names now.